tests/cypress/features/common/common.js

```diff
@@ -71,24 +71,9 @@ Given(/^I am on (.*?) page$/, function (page) {
   page = page.toLowerCase();
   cy.server();
   switch (page) {
-    // case 'dashboard':
-    //   cy.visit(urls.dashboard);
-    //   break;
     case 'register delegate':
       cy.visit(urls.registerDelegate);
       break;
-    // case 'delegates':
-    //   cy.visit(urls.delegates);
-    //   break;
-    // case 'wallet':
-    //   cy.visit(urls.wallet);
```

Comment:
please rever this commented they are commented becouse of failing we need to recover them some time later

Reply:
Actually I removed them because they were wasteful case statements. 
The default case is capable of handling the above scenarios 

since the default case had

` cy.visit(urls[page])`